spec/jobs/bgs/submit_form674_job_spec.rb

```diff
@@ -6,30 +6,36 @@
   let(:user) { FactoryBot.create(:evss_user, :loa3) }
   let(:dependency_claim) { create(:dependency_claim) }
   let(:all_flows_payload) { FactoryBot.build(:form_686c_674_kitchen_sink) }
+  let(:birth_date) { '1809-02-12' }
   let(:vet_info) do
     {
       'veteran_information' => {
-        'birth_date' => '1809-02-12',
         'full_name' => {
-          'first' => 'WESLEY', 'last' => 'FORD', 'middle' => nil
+          'first' => 'WESLEY', 'middle' => nil, 'last' => 'FORD'
```

Comment:
Nitpick/non blocking: maybe consider `Faker` from some of the mock data in this file

Reply:
for this I went off of precedent in the file, could look at Faker in the future.